test/ByteCode/rlexe.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<regress-exe>
+  <test>
+    <default>
+      <tags>exclude_dynapogo,exclude_amd64,exclude_arm,require_backend</tags>
+      <compile-flags>-GenerateLibraryByteCodeHeader</compile-flags>
```

Comment:
><compile-flags>-GenerateLibraryByteCodeHeader</compile-flags> [](start = 6, length = 61)

So the plan is to use a test to generate the entire bytecode and check for differences, and then alert the user to the fact that the bytecode needs to be updated?

Since we've already got the binaries handy, this is a good approach.

I think right now the way we regen bytecode depends on scripts. Since we're talking about eventually replacing rl.exe anyway, I don't want to suggest that we put that functionality into rl.exe, but we should define a unified way to generate bytecode.